src/components/WebexVideoSettings/WebexVideoSettings.scss

```diff
@@ -0,0 +1,26 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-video-settings {
+  position: relative;
+  .camera-warning {
+    color: $md-red-60;
+    display: flex;
+    justify-content: space-between;
```

Comment:
does this mean that the extra space will be inserted between the icon and the text?